README.md

```diff
@@ -11,23 +11,39 @@ It is a very small version of a full Snowplow data collection pipeline: small en
 
 ## Quick start
 
-Snowplow Micro is configured at runtime through the [Stream Collector][stream-collector] configuration and the [Iglu][iglu] resolver configuration. So, to start Snowplow Micro:
-
-1. Update the [stream collector configuration][stream-collector-config] for Snowplow Micro: [example configuration][collector-config-example].
-2. Update the [Iglu resolver configuration][iglu-resolver-config] for Snowplow Micro: [example configuration][iglu-resolver-example].
-3. Run Micro either using Docker or using Java.
-
 ### Using Docker
 
 Snowplow Micro is hosted on Docker Hub ([snowplow/snowplow-micro][docker-micro]) with images available for both `amd64` and `arm64` architectures.
 
+The following command can be used to run Micro with the default configuration:
+
+```bash
+docker run -p 9090:9090 snowplow/snowplow-micro
```

Comment:
I suggest put the `:1.3.0` tag even in this minimal example.  Bad things could happen to unwitting users if we change config or api format in a future `latest` version.